realtimenet/display.py

```diff
@@ -133,6 +134,47 @@ def display(self, img, display_data):
         return img
 
 
+class DisplayFPS(BaseDisplay):
+    """
+    Display camera fps and inference engine fps on debug window.
+    """
+
+    def __init__(self, neural_network, y_offset=20):
+        super().__init__(y_offset)
+        self.net = neural_network
+
+        self.fps_update_rate = 0.1
+        self.running_delta_time_inference = 1. / (self.net.fps / self.net.step_size)
+        self.running_delta_time_camera = 1. / self.net.fps
+        self.last_update_time_camera = time.perf_counter()
+        self.last_update_time_inference = time.perf_counter()
+
+    def display(self, img: np.ndarray, display_data: dict) -> np.ndarray:
+
+        now = time.perf_counter()
+        if display_data['prediction'] is not None:
+            # Inference engine frame rate
+            delta = (now - self.last_update_time_inference)
+            self.running_delta_time_inference += self.fps_update_rate * (delta - self.running_delta_time_inference)
+            self.last_update_time_inference = now
+        inference_engine_fps = 1. / self.running_delta_time_inference
+
+        # Camera FPS counting
+        delta = (now - self.last_update_time_camera)
+        self.running_delta_time_camera += self.fps_update_rate * (delta - self.running_delta_time_camera)
+        camera_fps = 1. / self.running_delta_time_camera
+        self.last_update_time_camera = now
+
+        # Text color change if inference engine fps go below certain range
+        text_color = (0, 255, 0) if inference_engine_fps > self.net.step_size * 0.75 else (0, 0, 255)
```

Comment:
Let's add a class attribute for this `0.75`. How about `self.low_performance_rate`?